src/platform/silabs/efr32/OTAImageProcessorImpl.cpp

```diff
@@ -30,6 +30,20 @@ extern "C" {
 
```

Comment:
If/when we decide to support multi-ota for Series 3 the same changes would need to be applied to src/platform/silabs/multi-ota/OTAFirmwareProcessor.cpp

Reply:
@selissia Yes that can be done in a subsequent pr. The series 3 bootloader and internal image update are still in flux. Once stabilized, we can apply the final state to the muti-ota